### ISSUE TITLE
Badly named single-letter variable names are not detected

### ISSUE THREAD

**Issue by ThiefMaster** [PyCQA/pycodestyle#341]
```shell
> cat test.py
def l(I):
    l = O = l = 0 + I
> pep8 test.py
>
```

> Never use the characters 'l' (lowercase letter el), 'O' (uppercase letter oh), or 'I' (uppercase letter eye) as single character variable names.

> In some fonts, these characters are indistinguishable from the numerals one and zero. When tempted to use 'l', use 'L' instead.

**Comment by IanLee1521** [comment]
For the record, this is from: https://www.python.org/dev/peps/pep-0008/#names-to-avoid
